README.md

```diff
@@ -62,18 +62,23 @@ Host: <host ip> # default: 0.0.0.0
 Port: <host port> # default: 8089
 
 [Microsoft Teams]
-Connector: <webhook url> # required value
+connector1: <webhook url> # At least one connector is required
```

Comment:
Why that keys are now in lowercase? :wink:

Reply:
Switched back to the initial capital letter :)